tasks/managed/marketplacesvm-push-disk-images/marketplacesvm-push-disk-images.yaml

```diff
@@ -44,6 +48,8 @@ spec:
             secretKeyRef:
               name: $(params.cloudMarketplacesSecret)
               key: key
+        - name: UPLOAD_CONTAINER_NAME
```

Comment:
I don't see `UPLOAD_CONTAINER_NAME` used anywhere in the script section, so what does this do?

Reply:
It's an environment variable for our tooling, it's being used here: https://github.com/release-engineering/pubtools-marketplacesvm/blob/main/src/pubtools/_marketplacesvm/cloud_providers/base.py#L21

Our goal is to change its name from `pubupload` to `konfluxupload` so the artifacts doesn't get mixed up. This is important for our CVE notification service called Impactwatch

Reply:
ack, I think a comment saying the script pulls it from the env could be nice, but not totally required

Reply:
Ok, I'll add it